plugins/modules/eval_nat_network_acls.py

```diff
@@ -33,7 +33,7 @@
     description:
     - The port range used by the AWS resource in your Amazon VPC you want to test connectivity from.
     type: str
-    required: false
+    required: true
```

Comment:
It should be optional, so required: false. Please update in the argument spec also.

Reply:
Removing the required field should work. I shall update the arg spec accordingly.